include/aws/cryptosdk/private/config.h.in

```diff
@@ -24,6 +25,19 @@
 // not on a tag, we'll add the git revision to the version strings
 #cmakedefine AWS_CRYPTOSDK_PRIVATE_GITVERSION "@AWS_CRYPTOSDK_PRIVATE_GITVERSION@"
 
+#ifndef AWS_CRYPTOSDK_PRIVATE_GITVERSION
```

Comment:
I'd prefer to avoid having anything in `config.h` proper that is not a cmake substitution. Can we move this stuff (and the #include for version.h) to some other header?

Reply:
Done.